fastlane/Fastfile

```diff
@@ -593,6 +593,37 @@ end
       SimCtl.create_device device, devicetype, runtime
     }
   end
+
+  #####################################################################################
+  # UI tests lanes
+  #####################################################################################
+
+    # Props to work done by others in this file and in
+    # https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/fastlane/Fastfile
+    # this is a combined rip-off from both
+
+    #####################################################################################
+    # run_UI_tests
+    # -----------------------------------------------------------------------------------
+    # Usage:
+    # bundle exec fastlane run_UI_tests [device:<Name of the iOS Simulator >]
+    #
+    # Example:
+    # bundle exec fastlane run_UI_tests device:"iPhone 12"
+    #####################################################################################
+  # A lane to run a subset of automated UI tests. Since only login tests are
+  # selected for now, there will be no credentials race condition.
```

Comment:
Nitpick: this could go as part of the fenced comment block above, e.g. like in the `code_freeze` lane: https://github.com/Automattic/simplenote-ios/blob/b7390955954e75215a6935c59fb831f680bd79da/fastlane/Fastfile#L53-L58

Reply:
Done! Thank you